Standardize return formats

JulianGriggs opened the pull request:
Addresses #2 
Copied from the commit message: 
An implementation detail worth noting is that I got rid of the notion
of last_status_code. It's only present use was for setting the 'code'
field in certain API actions. Saving that information to an instance
variable was unnecessary and when it's possible to avoid having state
in a service, I think it is good to do so.

Commit:
Standardize API return format to array with 'code' and 'data' fields

An implementation detail worth noting is that I got rid of the notion
of last_status_code. It's only present use was for setting the 'code'
field in certain API actions. Saving that information to an instance
variable was unecessary and when it's possible to avoid having state
in a service, I think it is good to do so.

Commit:
Update Readme for new return values

talk2cerlin commented:
Good work.

FYI the layer which communicates with Akamai server is taken from [Raben's Plugin](https://github.com/raben/Akamai/), that is the reason why it wasn't modified.

Looks like the `Usage sample` from the readme.md is not updated with the latest changes? Can you do it?

Commit:
Update usage example in readme

- Update for new return formats
- Remove var_dumps in favor of phpdoc for types
- Use php markdown for syntax highlighting

JulianGriggs commented:
Thanks, and good catch! I had totally forgotten to update that (but now I have). 

Yeah I noticed that it came from the original plugin but as that repo hasn't been updated in about a year, and I was already using this one, it made more sense to update it here.

talk2cerlin commented:
Sorry for the delay. I was busy the whole week.
